week-5/ghostdb/Scripts/1_script_create_tables.sql

```diff
@@ -0,0 +1,40 @@
+create TABLE customers(
+  customer_id serial PRIMARY KEY,
+  customer_name text,
+  customer_city text,
+  customer_state text,
+  customer_country text
```

Comment:
Another good option is to just use varchar and limit the size for name, city, state, and country. Some troll users could enter a text of 2^10 characters.

Reply:
thanks for the recommendation.

Reply:
💯